testsuite/features/support/env.rb

```diff
@@ -26,13 +26,14 @@
 STARTTIME = Time.new.to_i
 Capybara.default_max_wait_time = ENV['CAPYBARA_TIMEOUT'] ? ENV['CAPYBARA_TIMEOUT'].to_i : 10
 DEFAULT_TIMEOUT = ENV['DEFAULT_TIMEOUT'] ? ENV['DEFAULT_TIMEOUT'].to_i : 250
+PROVIDER = ENV["PROVIDER"]
```

Comment:
```suggestion
$provider = ENV["PROVIDER"]
```

Reply:
I suggest that you add a function like this at the end of `commonlin.rb` and you use it everywhere where you have these comparisons:


```ruby
def is_aws_provider?
  $provider.include? 'aws'
end
```